examples/Basic/package.json

```diff
@@ -4,19 +4,42 @@
 	"private": true,
 	"scripts": {
 		"start": "node node_modules/react-native/local-cli/cli.js start",
-		"test": "jest"
+		"test": "jest",
+		"postinstall": "rm -rf node_modules/react-native-blur/.git node_modules/react-native-blur/examples/",
+		"web:build:vendor-dev": "rm -rf web/vendor-dev; NODE_ENV=development webpack --config web/vendor.webpack.config.js || rm -rf web/vendor-dev",
```

Comment:
Is it required to have all these commands to run a simple react-native-web example? Probably we can shrink them somehow? A bit overwhelming...